install_config/persistent_storage/storage_classes.adoc

```diff
@@ -0,0 +1,212 @@
+[[install-config-persistent-storage-storage-classes]]
+= Dynamic Provisioning and Creating Storage Classes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+The *StorageClass* resource object is used to describe and classify storage that can be requested, as well as 
+provide a means for passing in parameters for *dynamically provisioned storage* on-demand.  StorageClasses can also serve
+as a management mechanism for controlling different levels of storage and access to the storage.  Cluster Administrators (cluster-admin)
+or Storage Administrators (storage-admin) need to define and create the StorageClasses that can then be requested by users without the users needing to have
+any intimate knowledge about the underlying storage volume sources. Additonal information and examples can be found using the links below:
+
+- xref:../../install_config/storage_examples/storage_classes_dynamic_provisioning.adoc#install-config-storage-examples-storage-classes-dynamic-provisioning[Examples and uses of StorageClasses for Dynamic Provisioning]
+
+- xref:../../install_config/storage_examples/storage_classes_legacy.adoc#install-config-storage-examples-storage-classes-legacy[Examples and uses of StorageClasses without Dynamic Provisioning] 
+
+The Kubernetes
+xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent volume]
+framework enables this functionality and allows administrators to provision a cluster with persistent storage
+and gives users a way to request those resources without having any knowledge of
+the underlying infrastructure.
+
+Many storage types are available for use as persistent volumes in
+{product-title}. While all of them can be statically provisioned by an
+administrator, some types of storage can be created dynamically using the built in provider and plug-in APIs.
+
+
+[[available-dynamically-provisioned-plug-ins]]
+== Available Dynamically Provisioned Plug-ins
+
+{product-title} provides the following _provisioner plug-ins_, which have
+generic implementations for dynamic provisioning that use the cluster's
+configured provider's API to create new storage resources:
+
+[options="header"]
+|===
+
+|Storage Type |Provisioner Plug-in Name
+
+|OpenStack Cinder
+|`kubernetes.io/cinder`
+
+|AWS Elastic Block Store (EBS)
+|`kubernetes.io/aws-ebs`
+
+|GCE Persistent Disk (gcePD)
+|`kubernetes.io/gce-pd`
+
+|GlusterFS
+|`kubernetes.io/glusterfs`
+
+
```

Comment:
Kube has the vSphere plugin as well: http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html